Master-OB-OpenAPI.json

```diff
@@ -15165,6 +15165,118 @@
         "Curtailed": {
           "label": "Curtailed",
           "description": ""
+        },
+        "Emergency": {
+          "label": "Emergency",
+          "description": ""
+        },
+        "Environment": {
+          "label": "Environment",
+          "description": ""
+        },
+        "Forced": {
+          "label": "Forced",
+          "description": ""
+        },
+        "Grid": {
+          "label": "Grid",
+          "description": ""
+        },
+        "Islanded": {
+          "label": "Islanded",
+          "description": ""
+        },
+        "Shutdown": {
+          "label": "Shutdown",
+          "description": ""
+        },
+        "Standby": {
+          "label": "Standby",
+          "description": ""
+        },
+        "Operational": {
+          "label": "Operational",
+          "description": ""
+        },
+        "UnderMaintenance": {
+          "label": "Under Maintenance",
+          "description": ""
+        },
+        "CommunicationFailure": {
+          "label": "Communication Failure",
+          "description": ""
+        },
+        "Decommissioned": {
```

Comment:
I think "Decommissioned" is a business state.

Reply:
So this shouldn't be an enum for OperationalStatusItemType but OperationalPhaseItemType?

Reply:
Yes, I think so